test-data/unit/check-errorcodes.test

```diff
@@ -925,3 +925,12 @@ def f(d: D, s: str) -> None:
 [case testRecommendErrorCode]
 # type: ignore[whatever]  # E: type ignore with error code is not supported for modules; use `# mypy: disable-error-code=...`  [syntax]
 1 + "asdf"
+
+[case testShowErrorCodesInConfig]
+# flags: --config-file tmp/mypy.ini
+# Test 'show_error_codes = True' in config doesn't raise an exception
+var: int = ""  # E: Incompatible types in assignment (expression has type "str", variable has type "int")  [assignment]
+
+[file mypy.ini]
+\[mypy]
+show_error_codes = True
```

Comment:
Do we need a test for `hide_error_code = False / True`?

Reply:
I've already added one in `check-flags.test` right at the end.